osf-packager/src/main/java/org/dataconservancy/cos/packaging/OsfContentProvider.java

```diff
@@ -98,140 +64,55 @@
         private static final Property RDF_TYPE = ResourceFactory.createProperty(Rdf.Ns.RDF, "type");
     }
 
-    private static final RelationshipResolver DEFAULT_RESOLVER = cxt.getBean("jsonApiRelationshipResolver", RelationshipResolver.class);
+    private Logger                  LOG = LoggerFactory.getLogger(OsfContentProvider.class);
+    private OsfPackageGraph         packageGraph;
+    private RelationshipResolver    resolver;
+    private Model                   domainObjects = null;
 
-    private RelationshipResolver resolver;
+    static final ClassPathXmlApplicationContext cxt = new ClassPathXmlApplicationContext(
+        "classpath*:org/dataconservancy/cos/osf/client/config/applicationContext.xml",
+        "classpath:/org/dataconservancy/cos/packaging/config/applicationContext.xml");
 
-    public IpmPackager() {
-        resolver = DEFAULT_RESOLVER;
-    }
-
-    public IpmPackager(RelationshipResolver resolver) {
-        this.resolver = resolver;
-    }
+    private static final String MISSING_PROVIDER = "missing_storage_provider";
 
-    public RelationshipResolver getResolver() {
-        return resolver;
+    /**
+     * Construct a content provider from the given graph and a default relationship resolver.
+     * @param graph The OSF package graph containing the package content.
+     */
+    public OsfContentProvider(OsfPackageGraph graph) {
+        packageGraph = graph;
```

Comment:
If the member variables (packageGraph, resolver) aren't `final`, then just delegate to `OsfContentProvider(OsfPackageGraph, Resolver)`?